simulator/risc_v/riscv_register/riscv_register.cpp

```diff
@@ -5,10 +5,11 @@
 
 #include "riscv_register.h"
 
-const RISCVRegister RISCVRegister::zero = RISCVRegister( RISCV_REG_zero);
+const RISCVRegister RISCVRegister::zero           = RISCVRegister( RISCV_REG_zero);
```

Comment:
From my experience, pretty alignments are quite useless — they pollute git history and patches, while introduce small value to the code.